### DISCUSSION TITLE
版本选择问题

### DISCUSSION THREAD

**Discussion by ownbylichaobao** [milvus-io/milvus#17281] [Q&A and General discussion]
请问线上使用 master分支会是比较好的选择么？相比较2.0.2 或者2.0.3来说

**Comment by yhmo** [comment]
master分支在持续开发中，相比较2.0.2 或者2.0.3来说修复了很多旧的bug，性能也有提升。但不是每个commit都经过完整测试，可能会有重大bug没有发现。

**Reply by ownbylichaobao** [reply]
好的 感谢

**Reply by xiaofan-luan** [reply]
可以等一下2.1，下周就会发布了，会有很大的性能提升？

**Reply by ownbylichaobao** [reply]
> 可以等一下2.1，下周就会发布了，会有很大的性能提升？

赞 期待

**Reply by gobigrassland** [reply]
本周会发布2.1吗？我们等着存储数据支持string类型呢

**Reply by yhmo** [reply]
> 本周会发布2.1吗？我们等着存储数据支持string类型呢

比较难，还有一些P0问题没解决。

**Reply by gobigrassland** [reply]
理解~那你们这边有个大致发布的时间点吗？

**Reply by yhmo** [reply]
> 理解~那你们这边有个大致发布的时间点吗？
本来预期是5月份发布的，现在看这个版本内容较多，相应的问题也多，要等这个[列表](https://github.com/milvus-io/milvus/issues?q=is%3Aopen+is%3Aissue+label%3Apriority%2Fcritical-urgent)里的问题修复得差不多了才敢发布。目前预计还要一周吧。

**Reply by gobigrassland** [reply]
期待中~
